notebooks/model_serving/ray-serve-llm.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # 1: Wrap the pretrained sentiment analysis model in a Serve deployment.
</pre><p>Should we change the name to <em>flan-t5 instruction model</em> instead of <em>sentiment analysis</em> model to avoid confusion?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>